ciphey/basemods/Checkers/quadgrams.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import re
+from math import log10
+from typing import Dict, Optional
+
+from ciphey.iface import Checker, Config, ParamSpec, T, Translation, registry
+from rich.logging import RichHandler
+
+
+@registry.register
+class Quadgrams(Checker[str]):
+
+    """
+    Uses Quadgrams to determine plaintext
+    """
+
+    def check(self, ctext: T) -> Optional[str]:
+        logging.debug("Trying Quadgrams checker")
+        ctext = re.sub("[^A-Z]", "", ctext.upper())
```

Comment:
Removes everything that isn't a letter? maybe a comment here?

Reply:
Yes it removes everything that's not a letter and capitalizes it so that it can check for the quadgrams. I'll add a comment.